packages/editor-ui/src/views/SettingsUsersView.test.ts

```diff
@@ -119,6 +119,7 @@ describe('SettingsUsersView', () => {
 	it('disables the invite button on SAML login', async () => {
 		const pinia = createTestingPinia({ initialState: getInitialState() });
 		const ssoStore = useSSOStore(pinia);
+		// @ts-expect-error updating a readonly property
 		ssoStore.isSamlLoginEnabled = true;
```

Comment:
not pretty. would be nice if we enabled this by setting the settings here instead.

Reply:
there is more to refactor here than what's in the scope of this PR. maybe another time.